utils/training_loop.py

```diff
@@ -13,42 +13,47 @@ def training_loop(experiment, env, agent, model):
         print('Only Inference')
         model = PPO.load(experiment.LOAD_WEIGHT_DIRECTORY)
         print(f"Loaded weights from {experiment.LOAD_WEIGHT_DIRECTORY} for inference.")
-
-    collision_counter, episode_counter, total_steps = 0, 0, 0
-    all_rewards, all_actions = [], []
-
-    for episode in range(experiment.EPOCHS):
-        print(f"@ Episode {episode + 1} @")
-        current_state = env.reset()
-        done = False
-        episode_sum_of_rewards, steps_counter = 0, 0
-        episode_counter += 1
-        resume_experiment_simulation(env)
-        actions_per_episode = []
-        while not done:
-            steps_counter += 1
-            total_steps += 1
-            action = agent.get_action(model, current_state, total_steps, experiment.EXPLORATION_EXPLOTATION_THRESHOLD)
-            current_state, reward, done, _ = env.step(action)
-            episode_sum_of_rewards += reward
-            if reward < experiment.COLLISION_REWARD or done:
-                pause_experiment_simulation(env)
+    else:
+        collision_counter, episode_counter, total_steps = 0, 0, 0
+        all_rewards, all_actions = [], []
+
+        for episode in range(experiment.EPOCHS):
+            print(f"@ Episode {episode + 1} @")
+            current_state = env.reset()
+            done = False
+            episode_sum_of_rewards, steps_counter = 0, 0
+            episode_counter += 1
+            resume_experiment_simulation(env)
+            actions_per_episode = []
```

Comment:
You define 'actions_per_episode' but never append actions to it. Is this intended?